src/components/__test__/Footer.test.js

```diff
@@ -0,0 +1,12 @@
+/* eslint-disable no-undef */
+import React from 'react';
+import { shallow } from 'enzyme';
+import Footer from '../Footer.js';
+
+const wrapper = shallow(<Footer />);
```

Comment:
The footer needs to be wrapped into the `BrowserRouter` or `MemoryRouter` when mounted by jest as it contain `Link` components